### PR TITLE
Random editing of ray tracing chapter

### PR DESCRIPTION
**Pull request by krOoze**

### CODE REVIEW

File: chapters/VK_NV_ray_tracing/raytracing-resources.txt [hunk 1]
```diff
@@ -126,11 +126,11 @@ flink:vkCmdBuildAccelerationStructureNV.
     pname:geometryCount must: be `0`
   * [[VUID-VkAccelerationStructureInfoNV-type-02426]]
     If pname:type is ename:VK_ACCELERATION_STRUCTURE_TYPE_BOTTOM_LEVEL_NV
-    then pname:instanceCount must: be `0`.
-  * [[VUID-VkAccelerationStructureInfoNV-compactedSize-02427]]
-    If pname:compactedSize is not `0` then both pname:geometryCount and
-    pname:instanceCount must: be `0`
```
Comment (krOoze) [inline chapters/VK_NV_ray_tracing/raytracing-resources.txt]: Already present in `VkAccelerationStructureCreateInfoNV` VU

File: chapters/VK_NV_ray_tracing/raytracing-resources.txt [hunk 2]
```diff
@@ -183,10 +181,6 @@ include::../../api/enums/VkBuildAccelerationStructureFlagBitsNV.txt[]
     memory and the final result build, potentially at the expense of build
     time or trace performance.
 
-ename:VK_BUILD_ACCELERATION_STRUCTURE_PREFER_FAST_TRACE_BIT_NV and
-ename:VK_BUILD_ACCELERATION_STRUCTURE_PREFER_FAST_BUILD_BIT_NV must: not be
-used in the same flags.
```
Comment (krOoze) [inline chapters/VK_NV_ray_tracing/raytracing-resources.txt]: Converted to VU in `VkAccelerationStructureInfoNV`.

File: chapters/VK_NV_ray_tracing/raytracing-resources.txt [hunk 3]
```diff
@@ -485,35 +479,6 @@ include::../../api/protos/vkBindAccelerationStructureMemoryNV.txt[]
     slink:VkBindAccelerationStructureMemoryInfoNV, describing images and
     memory to bind.
 
-.Valid Usage
-****
```
Comment (krOoze) [inline chapters/VK_NV_ray_tracing/raytracing-resources.txt]: All duplicates of `VkBindAccelerationStructureMemoryInfoNV` VUs.

File: chapters/VK_NV_ray_tracing/raytracing-resources.txt [hunk 4]
```diff
@@ -546,23 +511,24 @@ include::../../api/structs/VkBindAccelerationStructureMemoryInfoNV.txt[]
...
-    pname:memoryOffset must: be an integer multiple of the pname:alignment
-    member of the sname:VkMemoryRequirements structure returned from a call
-    to fname:vkGetAccelerationStructureMemoryRequirementsNV with
-    pname:accelerationStructure
-  * [[VUID-VkBindAccelerationStructureMemoryInfoNV-size-02454]]
-    The pname:size member of the sname:VkMemoryRequirements structure
+    slink:VkMemoryRequirements structure returned from a call to
+    flink:vkGetAccelerationStructureMemoryRequirementsNV with
+    pname:accelerationStructure and pname:type of
+    ename:VK_ACCELERATION_STRUCTURE_MEMORY_REQUIREMENTS_TYPE_OBJECT_NV
```
Comment (krOoze) [inline chapters/VK_NV_ray_tracing/raytracing-resources.txt]: added this enum requirement, which I feel should be there. same below

File: chapters/VK_NV_ray_tracing/raytracing-resources.txt [hunk 5]
```diff
@@ -165,8 +165,6 @@ include::../../api/enums/VkBuildAccelerationStructureFlagBitsNV.txt[]
   * ename:VK_BUILD_ACCELERATION_STRUCTURE_ALLOW_UPDATE_BIT_NV indicates that
     the specified acceleration structure can: be updated with pname:update
     of ename:VK_TRUE in flink:vkCmdBuildAccelerationStructureNV.
-    This flag must: only be set on a build that is not an update or is an
-    update with a source that also had this flag set.
```
Comment (krOoze) [inline chapters/VK_NV_ray_tracing/raytracing-resources.txt]: Already the VU's say all of it incl this flag **must** be identical. This sentence IMO only confuses.

**Comment by dgkoch** [comment]
@oddhack these changes look good, I'll let you merge them so I don't mess up your workflow!

**Pull request merged**